install.py

```diff
@@ -93,6 +105,8 @@ def waitForContentPath():
     # These keys aren't created until studio closes, so keep retrying until they exist
 
     def func():
+        requestKillStudioProcess()
+        time.sleep(5)
```

Comment:
Minor point - but rather than wait for five seconds, can we poll for the key up to a maximun of N seconds?

Reply:
That's doable. I mainly chose this way because it was simpler. It follows the pattern of sleeping 5 seconds that already was being used previously, and required fewer edits to existing functionality. In addition, we should avoid repeatedly sending the taskkill signal.

Reply:
I included the 5 second sleep here mainly to avoid the repeated sending of the signal.